nimble/include/nimble/hci_common.h

```diff
@@ -552,9 +552,17 @@ extern "C" {
 
 /* --- LE set periodic advertising parameters (OCF 0x003E) */
 #define BLE_HCI_LE_SET_PER_ADV_PARAMS_LEN           (7)
```

Comment:
Looks like we also used PER_ADV, so probably this is why you use this pattern for functions name. Can we chance all this defines to FOO_PERIODIC_ADV_FOO?